src/styles/_variables.scss

```diff
@@ -15,8 +15,8 @@ $shr-context-line: #5EBFFF;
 $shr-context-light: #E5F7FF;
 
 // Custom variables
-$highlight-yellow: rgb(255, 255, 70);
-$highlight-selected-yellow: rgb(255, 210, 5);
+$highlight-yellow: rgb(255, 255, 5);
```

Comment:
[DL1]  Should this be be `rgb(255, 255, 3)`?  Thats what I see in the attachment in the JIRA.